lib/algo/memoization/test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const knapsack = require('./0-1 Knapsack');
+
+class Item {
+  constructor(profit, volume) {
+    this.profit = profit;
+    this.volume = volume;
+  }
+}
+
+const expect = chai.expect;
+
+// var items = [...new Array(9)].map(() => new Item(Math.round(Math.random() * 10) + 1, Math.round(Math.random() * 8) + 1))
+describe('knapsack', () => {
+  const items = [
+    new Item(12, 2),
+    new Item(5, 4),
+    new Item(1, 5),
+    new Item(56, 23),
+    new Item(2, 6),
+    new Item(1, 5),
+    new Item(12, 8),
+    new Item(2, 3)
+  ];
+  const capacity = 7;
+  it('should solve knapsack', done => {
```

Comment:
Can there be a better title, for persons who don't know `knapsack`, it would be difficult to follow. Something like
`should return maximum total value for capacity 7`